### ISSUE TITLE
Docker make my macbook pro too hot that burnt its mainboard.

### ISSUE THREAD

**Issue by datntvn** [docker/for-mac#5285]
Last month (December 2020), I used mac book pro 2017, 15 inch.

Updated to Latest version of Docker.

I'm using Docker, with 2 containers are running
 1. Mysql server version 8
 2. Another container for VNC connection

It was too hot that burnt my mac book pro's mainboard.

I've lost all my work including data for it. At least Docker team should provide any mechanism to prevent this from happening.

**Comment by pierreminik** [comment]
I don't know a lot about Docker but shouldn't overheating be (and isn't it already) built into macOS/the Macbook? I can't think of any reason a software should be able to fry its hardware... How do you know it wasn't hardware malfunction that caused this?

**Comment by kuberlog** [comment]
Docker burnt out my macbook charger this morning. Granted, I was using an underpowered charger, but it worked just fine all last week. For some reason, idling postgres in a docker container tipped it over the edge.

**Comment by datntvn** [comment]
@pierreminik 

That was in the morning, I turned on my Macbook for about 1 hour.
It was clearly that I opened Google Chrome with 2 web sites, Docker was run at startup with Mysql 8 container.
Other than that, I run another container which support VNC. 
Especially, on that VNC container I run a small OpenGL program.
I can easily recognize that, whenever I run that OpenGL program, the Macbook's fan automatically runs extremely loud.
Then, I locked screen the laptop, after 3 minutes, it was burned.

P/S: My team mates had refused to use Docker because whenever they open the Docker (without any container), the fan begins to run loudly.
